modules/appeals_api/app/serializers/appeals_api/base_evidence_submission_serializer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class BaseEvidenceSubmissionSerializer
```

Comment:
This is a new serializer based on `JSONAPI::Serializer` to handle evidence submissions in the segmented APIs - the [old, existing `AppealsApi::EvidenceSubmissionSerializer` for decision reviews](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/serializers/appeals_api/evidence_submission_serializer.rb#L4) did not use `JSONAPI::Serializer`